Production/nginx.tmpl

```diff
@@ -110,7 +110,31 @@
 	}
 		{{ end }}
 
-		
+  {{ if (eq $serviceName "lnd_lit") }}
+  location /lit {
+    proxy_set_header Host               $host;
+    proxy_set_header X-Real-IP          $remote_addr;
+    proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto  $scheme;
+    proxy_set_header Accept-Encoding "";
+    proxy_set_header Origin http://$lnd_backend;
+    proxy_set_header Connection $lnd_connection;
+    proxy_pass http://lnd_lit:8080/;
+  }
+
+	location ~* ^/lnrpc.Lightning/ {
+    proxy_pass http://lnd_lit:8080;
```

Comment:
To make sure this still works for normal gRPC requests that should go to `lnd`, this needs to be `https://$lnd_backend`, here and below.